qutebrowser/browser/commands.py

```diff
@@ -682,7 +682,10 @@ def _yank_url(self, what):
         else:
             flags |= QUrl.FullyEncoded
         url = QUrl(self._current_url())
-        url_query = QUrlQuery(url)
+        url_query = QUrlQuery()
+        if ';' in url.query():
+            url_query.setQueryDelimiters('=', ';')
```

Comment:
What if the query part contains both `&` and `;` for some reason? Ampersand is standard, while semicolon is only [recommended](http://www.w3.org/TR/html401/appendix/notes.html#h-B.2.2), so I think it should be handled like this:
```python
if '&' not in url.query():
    url_query.setQueryDelimiters('=', ';')
```